### ISSUE TITLE
Oracle Database 11g Release 2 support

### ISSUE THREAD

**Issue by renatakovacs** [djavorszky/ddn#176]
Can you please add the Oracle Database 11g Release 2 (latest version?) as an option too?

LR 6.2 and DXP support it.

Thanks,
Reni

**Comment by sorin-pop** [comment]
Hi Reni, 

For this, we would need such a version of Oracle server be set up and running somewhere on a central server in the office. 

This would be initially a task for the IT team, to identify the hardware resources needed and install and set up such an Oracle server.
Once this is done, I would only need to make some minor additional preparations in order to make that server work together with and be seen by CloudDB.

So I guess the ticket referenced by @djavorszky will take care of that initial IT department task.

Please note that the current Oracle 12c in the cloud is actually able to import dumps made from 11g Release 2 databases. So if only having the data is important, this should do it for the moment. But if for your use case is very relevant and important that an 11g Release 2 database is running under your portal, then indeed, the current one is not suitable. We'll do our best to have at least an additional Oracle server (with the version you requested) in the cloud, that would be a more ideal situation indeed. But I guess this will take some time. Let's see what will happen on https://github.com/djavorszky/ddn/issues/171 ...

**Comment by renatakovacs** [comment]
Thanks Sorin for the fast update. There isn't any particular case at this moment, however it came up couple of times in the past. Thanks for the import tip.

**Comment by sorin-pop** [comment]
This is low priority, as the current 12.1 can import 11.2 dumps. Migrated to https://github.com/djavorszky/liferay-support-ee/issues
